backend/src/lib/handlers/now-playing/index.js

```diff
@@ -0,0 +1,19 @@
+import HttpService from '../../services/http'
+
+const NowPlaying = {
+  addTrack: (track) => {
```

Comment:
Maybe something like `setCurrentTrack` might be a better name? (`addTrack` sounds like it's adding something to a list)

Reply:
I think `addTrack` is ok - because it's talking to an external service's `/add_track` endpoint which does just add something to a list (db). We happen to call this as a side effect of a new track starting & later query it to get /now_playing.